transaction/core/src/constants.rs

```diff
@@ -24,11 +24,14 @@ pub const MAX_TOMBSTONE_BLOCKS: u64 = 100;
 /// mobilecoins (MOB).
 pub const TOTAL_MOB: u64 = 250_000_000;
 
+/// one microMOB = 1e6 picoMOB
+pub const MICROMOB_TO_PICOMOB: u64 = 1_000_000;
+
 /// one milliMOB = 1e9 picoMOB
 pub const MILLIMOB_TO_PICOMOB: u64 = 1_000_000_000;
 
 /// Minimum allowed fee, denominated in picoMOB.
-pub const MINIMUM_FEE: u64 = 10 * MILLIMOB_TO_PICOMOB;
+pub const MINIMUM_FEE: u64 = 400 * MICROMOB_TO_PICOMOB;
```

Comment:
worth leaving a comment noting fee change?
```
/// fee change: v0 [0.01 MOB] -> v1 [0.0004 MOB]
```

Reply:
I dunno, we have git... :-)

Reply:
Guess my testimony as 'crypto historian' is - it's way easier to figure out what happened in the past if it's explicitly documented.

Reply:
It is explicitly documented, in the commit messages.